emmaa/priors/gene_list_prior.py

```diff
@@ -52,14 +54,17 @@ def make_search_terms(self, drug_gene_stmts=None):
         self.search_terms = terms
         return terms
 
-    def make_gene_statements(self):
+    def make_gene_statements(self) -> List[EmmaaStatement]:
         """Generate Statements from the gene list."""
+        if self.stmts:
+            return self.stmts
         drug_names = [st.name for st in self.search_terms if
                       st.type == 'drug']
-        indra_stmts = get_stmts_for_gene_list(self.gene_list, drug_names)
-        estmts = [EmmaaStatement(stmt, datetime.datetime.now(), [])
-                  for stmt in indra_stmts]
-        self.stmts = estmts
+        self.stmts = [
+            EmmaaStatement(stmt)
```

Comment:
Note before merging, I think this relies on #266

Reply:
Yup that's noted in the PR text... I went a bit overboard in this PR and wanted to change it back to draft but it doesn't seem possible at the moment